mecenv

```diff
@@ -13,6 +13,6 @@ unset LD_LIBRARY_PATH
 source "${CONDA_BASE}/etc/profile.d/conda.sh"
 conda activate "${CONDA_ENVNAME}"
 HERE=`dirname $(readlink -f $0)`
-export PYTHONPATH="${HERE}:${HERE}/dev/devpath:/reg/g/pcds/pyps/apps/dev/pythonpath:${HERE}/../common/dev/devpath"
+export PYTHONPATH="${HERE}:${HERE}/dev/devpath:/reg/g/pcds/pyps/apps/dev/pythonpath:${HERE}/../common/dev/devpath:${HERE}/dev/pulseshaping/mecps"
```

Comment:
cool, does this allow Eric to add his pulse shaping scripts to the dev environment?

Reply:
I just made a folder in the mec/dev area and cloned his repo in there. I am still *extremely* skeptical of this code, but Eric really wants it in. At least he has it version controlled now. It will be an iterative process to make that code kosher.